TaskLayer/MetaMorpheusTask.cs

```diff
@@ -216,7 +216,12 @@ public MyTaskResults RunTask(string output_folder, List<DbForTask> currentProtei
         {
             StartingSingleTask(displayName);
 
-            var tomlFileName = Path.Combine(output_folder, GetType().Name + "config.toml");
+            var path = output_folder;
+            if (!output_folder.Contains("netcoreapp2.0") && !(output_folder == ""))
```

Comment:
how much does this break if you don't check for this? I'm not really a fan of different code for unit tests vs. "normal" runs